lib/styles/components/blog-post.less

```diff
@@ -41,6 +41,12 @@
                 line-height: 1em;
                 border: none;
                 border-radius: 0;
+                a {
+                    svg {
+                        width: 28px;
```

Comment:
I know @ozanilbey hates px. @ozanilbey how can we revisit this without the use of px?

Reply:
@esokullu changed it to `width: 1.2em;`